templates/quickstart-crowd-dc.template.yaml

```diff
@@ -996,24 +976,14 @@ Resources:
       MaxSize: !Ref ClusterNodeMax
       MinSize: !Ref ClusterNodeMin
       TargetGroupARNs: [!Ref MainTargetGroup]
-      VPCZoneIdentifier: !Split
-        - ","
-        - Fn::ImportValue: !Sub "${ExportPrefix}PriNets"
+      VPCZoneIdentifier: !Split [ ",", !ImportValue ATL-PriNets]
```

Comment:
Why not use the `ExportPrefix`?

Reply:
This was me trying to tidy things up, although I went a bit too far :| I'll update things so that `ExportPrefix` is utilised.